core/ir/x86/decode_table.c

```diff
@@ -4604,10 +4604,9 @@ const instr_info_t prefix_extensions[][12] = {
     {INVALID, 0xf20f7337, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
   }, /* prefix extension 103 */
   {
-    {REX_B_EXT,  0x900000, "(rex.b ext 0)", xx, xx, xx, xx, xx, no, x, 0},
+    {REX_B_EXT,  0x900000, "(rex.b ext 0)", xx, xx, xx, xx, xx, no, x, END_LIST},
     {OP_pause,0xf3900000, "pause", xx, xx, xx, xx, xx, no, x, END_LIST},
-    /* we chain these even though encoding won't find them */
-    {OP_nop, 0x66900000, "nop", xx, xx, xx, xx, xx, no, x, tpe[103][3]},
+    {REX_B_EXT, 0x900000, "(rex.b ext 0)", xx, xx, xx, xx, xx, no, x, END_LIST},
```

Comment:
I would keep the 0x66 in the opcode field as it makes it easier to understand which row this is.

Reply:
Adding 0x66 to the opcode field will trigger the logic at https://github.com/DynamoRIO/dynamorio/blob/30da7c1edd19fac69ef61334a90fb5a4df0d17c9/core/ir/x86/decode.c#L916 and result in wrong operand sizes.